src/main/java/com/collections/ConcurrentCollectionsExample.java

```diff
@@ -0,0 +1,24 @@
+package com.collections;
+import java.util.concurrent.*;
+
+public class ConcurrentCollectionsExample {
+    public static void main(String[] args) {
+        // Concurrent List
+        CopyOnWriteArrayList<String> concurrentList = new CopyOnWriteArrayList<>();
+        concurrentList.add("A");
+        concurrentList.add("B");
+        System.out.println("Concurrent List: " + concurrentList);
+
+        // Concurrent Set
+        CopyOnWriteArraySet<String> concurrentSet = new CopyOnWriteArraySet<>();
+        concurrentSet.add("X");
+        concurrentSet.add("Y");
+        System.out.println("Concurrent Set: " + concurrentSet);
+
+        // Concurrent Map
```

Comment:
What is the difference between hashmap and concurrent hashmap.

In which case you will get concurrent modification exception